modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -1,22 +1,77 @@
 import * as bs58 from 'bs58';
 import * as crypto from 'crypto';
 import * as openpgp from 'openpgp';
-import Eddsa, { KeyShare } from '@bitgo/account-lib/dist/src/mpc/tss';
+import Eddsa, { KeyShare, JShare, SignShare, PShare, XShare, RShare, GShare } from '@bitgo/account-lib/dist/src/mpc/tss';
 
 import { BaseCoin, KeychainsTriplet } from '../baseCoin';
 import { Keychain } from '../keychains';
 import { BitGo } from '../../bitgo';
+import { KeyIndices, Memo, VerificationOptions, Wallet } from '..';
+import { RequestTracer } from './util';
+import _ = require('lodash');
+
+// #region Interfaces
+interface PrebuildTransactionWithIntentOptions {
+  reqId: RequestTracer;
+  intentType: string;
+  sequenceId?: string;
+  recipients: {
+    address: string;
+    amount: string | number;
+  }[];
+  comment?: string;
+  memo?: Memo;
+  verification?: VerificationOptions;
+}
+
+enum ShareKeyPosition {
+  USER = '1',
+  BACKUP = '2',
+  BITGO = '3',
+}
+
+// complete with more props if neccesary
+interface TxRequest {
+  txRequestId: string;
+  unsignedTxs: {
+    serializedTx: string;
+    signableHex: string;
+  }[];
+  signatureShares?: SignatureShareRecord[];
+}
+
+enum SignatureShareType {
+  USER = 'user',
+  BACKUP = 'backup',
+  BITGO = 'bitgo',
+}
+
+export interface SignatureShareRecord {
```

Comment:
do we need to export this?

Reply:
Im using it in the unit tests https://github.com/BitGo/BitGoJS/pull/1966/files#diff-2cef92cfdc88be0400ccdbd5d0b082064f0d66e8bbd01aa9ac4efa43ee2d9415R244